### DISCUSSION TITLE
What is the difference between Fuel Supply and Refueling Station

### DISCUSSION THREAD

**Discussion by steve-anthony-lee** [Opinionated-Blueprints/10-Books-Full-of-Rails#103] [General]
.. and do we need both?

I see the input belt on both of them and am curious what's you idea with both?

**Comment by brandon-gradient** [comment]
> .. and do we need both?

yes

https://github.com/Opinionated-Blueprints/10-Books-Full-of-Rails/wiki/Refuelling#fuel-supply
https://github.com/Opinionated-Blueprints/10-Books-Full-of-Rails/wiki/Refuelling#refuelling-station
This should explain it.

**Reply by steve-anthony-lee** [reply]
But if the refueling station has the belt on the side that's going into a box, why not just belt your fuel into it?

**Reply by Bocian-1** [reply]
> But if the refueling station has the belt on the side that's going into a box, why not just belt your fuel into it?

That's used for taking out fuel from a locomotive if it is worse than what the station is trying to load it with.
The supply train takes it back to the fuel provider on next visit
